src/ui/ResultsPopup.cpp

```diff
@@ -1,9 +1,51 @@
 #include "ResultsPopup.hpp"
 #include <managers/GuessManager.hpp>
 
+static int getLevelDifficulty(GJGameLevel* level) {
```

Comment:
i think adding this function to GuessManager would be better rather than copy/pasting it everywhere